### PR TITLE
Add configuring proxy for PCF platform monitoring tile section

### PR DESCRIPTION
**Pull request by okilker**
@jeffaholmes Please review. This is a quick draft from the JIRA. Not yet ready to be merged to master.

For PCF customers who use a proxy for all communication from PCF to external internet, proxy config must be added to allow two PCF apps deployed by tile to communicate with SaaS controller, otherwise KPI metrics, HRs and dashboards wont appear in controller.

### CODE REVIEW

File: docs-content/installing.html.md.erb [hunk 1]
```diff
@@ -69,6 +69,39 @@ $ cf curl /v2/info
   * **Username for Reporting Custom Dashboard**: Username of an account with custom dashboard creation permissions.
   * **Password**: Password for the above Account.
 
+## <a id='configure-proxy-settings'></a> Configuring Proxy Settings for the AppDynamics Platform Monitoring Tile
+
+If you're using a Proxy Server and SaaS controller, the two PCF apps deployed by the AppDynamics Platform Monitoring for PCF tile, AppDynamics Nozzle and the AppDynamics KPI Dashboard, require that you set environment variables to define the proxy host and port so the apps can communicate with the configured SaaS controller. 
+
+The following instructions assume the use of the Cloud Foundry CLI. You could also set the environment variables from the Apps Manager as well.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: This sentence can be removed: **You could also set the environment variables from the Apps Manager as well.**

File: docs-content/installing.html.md.erb [hunk 2]
```diff
@@ -69,6 +69,39 @@ $ cf curl /v2/info
...
+    $ cf set-env appdpcfdashboard-<version> HTTP_PROXY http://user@password:host:port/
+    $ cf set-env appdpcfdashboard-<version> HTTPS_PROXY https://user@password:host:https_port/
+ 
+ Restage the apps for the environment variables to take effect:
+
+    $ cf restage appdnozzle-<version>
+    $ cf restage  appdpcfdashboard-<version>
+
+## Apps Manager Configuration
+Set the environment variables from the Apps Manager as well.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Heading and instruction on how to set the environment variables from the Apps Manager.

File: docs-content/installing.html.md.erb [hunk 3]
```diff
@@ -69,6 +69,39 @@ $ cf curl /v2/info
...
+ 
+    $ cf set-env appdpcfdashboard-<version> HTTP_PROXY http://user@password:host:port/
+    $ cf set-env appdpcfdashboard-<version> HTTPS_PROXY https://user@password:host:https_port/
+ 
+ Restage the apps for the environment variables to take effect:
+
+    $ cf restage appdnozzle-<version>
+    $ cf restage  appdpcfdashboard-<version>
+
+## Apps Manager Configuration
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Configure the Apps Manager from the UI ?

File: docs-content/installing.html.md.erb [hunk 4]
```diff
@@ -69,6 +69,39 @@ $ cf curl /v2/info
   * **Username for Reporting Custom Dashboard**: Username of an account with custom dashboard creation permissions.
   * **Password**: Password for the above Account.
 
+## <a id='configure-proxy-settings'></a> Configuring Proxy Settings for the AppDynamics Platform Monitoring Tile
+
+If you're using a Proxy Server and SaaS controller, the two PCF apps deployed by the AppDynamics Platform Monitoring for PCF tile, AppDynamics Nozzle and the AppDynamics KPI Dashboard, require that you set environment variables to define the proxy host and port so the apps can communicate with the configured SaaS controller.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: Ideally break this into 3 shorter declarative sentences

File: docs-content/installing.html.md.erb (hunk 2 above)
Comment (jeffaholmes) [reply docs-content/installing.html.md.erb]: @okilker I can provide some additional info tomorrow am for Apps Manager option

**Comment by pavankrish123** [comment]
Looks Good. Thanks @okilker

File: docs-content/installing.html.md.erb (hunk 2 above)
Comment (okilker) [reply docs-content/installing.html.md.erb]: 👍 Awesome, thanks!

File: docs-content/installing.html.md.erb [hunk 5]
```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
 
 If you're using a Proxy Server and SaaS controller, the two PCF apps deployed by the AppDynamics Platform Monitoring for PCF tile, AppDynamics Nozzle and the AppDynamics KPI Dashboard, require that you set environment variables to define the proxy host and port so the apps can communicate with the configured SaaS controller. 
 
-The following instructions assume the use of the Cloud Foundry CLI. You could also set the environment variables from the Apps Manager as well.
+You can configure the proxy environment variables in either in  `filename.here` or using the Apps Manager.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: @jeffaholmes, In what file(s) do they enter their environment variables?

File: docs-content/installing.html.md.erb [hunk 6]
```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
...
+   ![](./images/restart-app-ui.png)
+
+### AppDynamics KPI Dashboard app 
+1. Navigate to the appdynamics space.
+1. Open the **Settings** tab for the `appdpcfdashboard-<version>` app. 
+1. Click **REVEAL USER PROVIDED ENV VARS**.
+
+    !!!! **Note:** Replace these Screenshots with KPI Dashboard App Images !!!!
+
+   ![](./images/reveal-user-envars.png)
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: 1 of 3, Need KPI Dashboard app image to replace Nozzle app

File: docs-content/installing.html.md.erb [hunk 7]
```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
...
+
+1. Add the proxy environment variables and click **SAVE**.
+
+   ![](./images/add-envars-save.png)
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: 2 of 3, Need KPI Dashboard app image to replace Nozzle app

File: docs-content/installing.html.md.erb [hunk 8]
```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
...
 
-    $ cf restage appdnozzle-<version>
-    $ cf restage  appdpcfdashboard-<version>
+1. Click the **Restart** button to restart the app.
 
-## Apps Manager Configuration
-Set the environment variables from the Apps Manager as well.
+   ![](./images/restart-app-ui.png)
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: 3 of 3, Need KPI Dashboard app image to replace Nozzle app

Comment (jeffaholmes) [reply docs-content/installing.html.md.erb]: @okilker I've added the 3 images to bottom of page: https://singularity.jira.com/wiki/spaces/~939476479/pages/589572732/Apps+Manager+instructions

File: docs-content/installing.html.md.erb (hunk 5 above)
Comment (jeffaholmes) [reply docs-content/installing.html.md.erb]: @okilker not aware of a file based option, only CF CLI (command line interface) and Apps Manager to set env var

File: docs-content/installing.html.md.erb (hunk 3 above)
Comment (jeffaholmes) [reply docs-content/installing.html.md.erb]: @okilker could you remove the trailing '/' from the KPI dashboard examples so it looks like this:
    $ cf set-env appdpcfdashboard-<version> HTTP_PROXY http://user@password:host:port
    $ cf set-env appdpcfdashboard-<version> HTTPS_PROXY https://user@password:host:https_port
There shouldn't be any need for them...thanks

Comment (okilker) [reply docs-content/installing.html.md.erb]: Done. ✅

**Comment by okilker** [comment]
@jeffaholmes @pavankrish123 @jknostman3 Can you confirm if this looks good to go?

File: docs-content/installing.html.md.erb [hunk 9]
```diff
@@ -75,7 +75,7 @@ You can configure the proxy environment variables in the Cloud Foundry Command L
 
 ### Using Cloud Foundry CLI to Set Proxy Environment Variables
 
-This section explains how to configure settings that use the Cloud Foundry CLI when interacting with the AppDynamics Nozzle and the AppDynamics KPI Dashboard, where `<version>` is the specific version of the tile deployed and substituted in the commands.
+This section explains how to configure settings using the Cloud Foundry CLI, where `<version>` is the specific version of the tile deployed and substituted in the commands.
```
Comment (okilker) [inline docs-content/installing.html.md.erb]: ✅

**Comment by pavankrish123** [comment]
looks good to me

**Comment by jncd** [comment]
@pavankrish123 and @okilker 
I've assumed that "Looks good to me" means okay to merge. If you weren't ready to have it merged, let me know and I'll revert.

Jake is away this week, so I'm doing the merge requests for him.

**Comment by pavankrish123** [comment]
Thanks @jncd ! We were expecting it to be merged. 

Pavan

**Pull request merged**